### ISSUE TITLE
Block the createdisk script if cert is not rotated for 30 days

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/snc#72]
Right now we don't have any check for cert rotation, we should first check if the cert rotated successfully for 30 days and then only start the disk creation process otherwise error out and tell user to wait for it. It will help our CI and also users who want to create the bundle using this script.

**Comment by gbraad** [comment]
Right, i wanted ot ask about this for the automated snc generation., You need to poll UNTIL the cert has rotated before the build can continue to package the images.

**Comment by gbraad** [comment]
@praveenkumar see https://bugzilla.redhat.com/show_bug.cgi?id=1736800#c9
```
oc get secret -n openshift-kube-apiserver -A -o json | jq -r '.items[] | select(.metadata.annotations."auth.openshift.io/certificate-not-after" | .!=null and fromdateiso8601<='$( date --date='+1year' +%s )') | "-n \(.metadata.namespace) \(.metadata.name)"' | xargs -n3 oc patch secret -p='{"metadata": {"annotations": {"auth.openshift.io/certificate-not-after": null}}}'
```

**Comment by anjannath** [comment]
When i tested this out, it didn't work for me, waited for 15-20mins and the certs were not rotated.

**Comment by praveenkumar** [comment]
Fixed #74
